launch/display.launch

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" ?>
+<launch>
+    <arg name="model" />
+	<param name="robot_description" command="$(find xacro)/xacro --inorder $(find robotont_description)/urdf/robotont_base.urdf.xacro"/>
+	<node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher"/>
+	<node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher"/>
```

Comment:
robot & joint state pubs are already started with the driver

Reply:
@Veix123 but this launch-file can be used standalone, without robotont robot

Reply:
@kruusamae Yes that's true, but we have to avoid closing the sate publishers which were automatically started within the bringup when we are working on a real robot... Currently, when the user executes display.launch for simple visualization of the robot and closes this window, one is not able to run any other demo without restarting the service (any new node with the same name will kill the previous instance). One solution could be an argument specifying whether we need to run the state publishers or we already have a simulated-, fake-, or real driver running somewhere in the system. We could also keep both nodes alive with anonymous names, but this will probably be a mess.

Reply:
ok. @mmaao rename current display.launch to display_simulated_robot.launch and then create another launch-file named display_real_robot.launch. in the second launch-file delete joint and robot state publishers